static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -348,8 +358,31 @@ define([
                 bonzo(player.el()).addClass(endState);
             });
         });
+
         player.on('playing', function () {
+            fader.addClass('is-faded');
             bonzo(player.el()).removeClass(endState);
+            bean.on(document.body, 'mousemove', setFaderTimeout);
+            bean.on(window, 'scroll', function (){
```

Comment:
I think it's better to use `mediator.on('window:throttledScroll', debounce(fn, 200));` instead if `scroll`.